README.md

```diff
@@ -0,0 +1,3 @@
+### Bonus Calculator Live Solve
+
```

Comment:
Nice addition. Maybe add a bit more detail about how it? And install instructions?